MergerService/Models/Task.cs

```diff
@@ -54,6 +54,7 @@ public void Print()
             }
             catch (Exception e)
             {
+                logger.LogError($"failed to deserialize task: {e.Message}");
```

Comment:
doesn't the logger gets exception as an argumnt

Reply:
then put the message "failed to deserialize task" and add the exception as an argument

Reply:
fine, even though it doesn't change the log at all (open telemetry + format limits)